docs/sphinx/source/whatsnew/v0.7.0.rst

```diff
@@ -99,6 +99,11 @@ API Changes
   The `weather` argument of the above methods is now the first, required
   positional argument and the `times` argument is kept as the second keyword
   argument for capability during the deprecation period.
+* The `effective_irradiance` argument for :py:func:`pvsystem.sapm` now requires
+  units of W/m^2. Previously, units for this input were suns. A deprecation
+  warning is raised if all `effective_irradiance` values are extremely low.
```

Comment:
revise warning type depending on resolution of comment below. revise 'extremely low' to something like "appear to be in suns (value < 2.0)"